ReadyPlayerOne/Views/Home/Index.cshtml

```diff
@@ -30,6 +30,7 @@
                 <li class="list-inline-item">
                     <a asp-action="Details" asp-route-id="@player.PlayerID">
                         <img src="~/images/@player.PlayerImage.Image" alt="@player.PlayerName"
+                            width="200" height="200"
```

Comment:
I just made the images smaller. You can choose to do this or not.

Reply:
This looks better. I will edit some of the prettiness as I tackle the larger aspects. Thank you again.